doc/source/core_composition.md

```diff
@@ -0,0 +1,98 @@
+---
+file_format: mystnb
+kernelspec:
+  name: geoutils
+---
+(core-composition)=
+
+# Composition from Rasterio and GeoPandas
+
+GeoUtils' main classes {class}`~geoutils.Raster` and {class}`~geoutils.Vector` are linked to [Rasterio](https://rasterio.readthedocs.io/en/latest/) and
+[GeoPandas](https://geopandas.org/en/stable/docs.html), respectively, through class composition.
+
+They directly rely on their robust geospatial handling functionalities, as well of that of [PyProj](https://pyproj4.github.io/pyproj/stable/index.html), and
+add a layer on top for interfacing between rasters and vectors with higher-level operations, performing easier numerical analysis, and adding more advanced geospatial functionalities.
+
+## The {class}`~geoutils.Raster` class composition
+
+The {class}`~geoutils.Raster` is a composition class with **four main attributes**:
+
+1. a {class}`numpy.ma.MaskedArray` as {attr}`~geoutils.Raster.data`,
+2. an {class}`affine.Affine` as {attr}`~geoutils.Raster.transform`
+3. a {class}`pyproj.crs.CRS` as {attr}`~geoutils.Raster.crs`, and
+4. a {class}`float` or {class}`int` as {attr}`~geoutils.Raster.nodata`.
+
+```{code-cell} ipython3
+:tags: [hide-output]
+
+import geoutils as gu
+
+# Initiate a Raster from disk
+raster = gu.Raster(gu.examples.get_path("exploradores_aster_dem"))
```

Comment:
Maybe as you did in the quick start section, it's good to split in two steps, so that people understand that Raster expects a string?
With beginners, the encapsulation was puzzling...

Reply:
Done everywhere consistently